packages/dd-trace/src/supported-configurations.json

```diff
@@ -62,6 +62,7 @@
     "DD_DBM_PROPAGATION_MODE": ["A"],
     "DD_DOGSTATSD_HOST": ["A"],
     "DD_DOGSTATSD_PORT": ["A"],
+    "DD_DYNAMIC_INSTRUMENTATION_CAPTURE_TIMEOUT_MS": ["A"],
```

Comment:
Should we map this to the older value for backwards compatibility and system-tests sake?